src/NUnitFramework/framework/Internal/InvalidDataSourceException.cs

```diff
@@ -24,7 +24,9 @@
 namespace NUnit.Framework.Internal
 {
     using System;
+#if !NETSTANDARD1_6
     using System.Runtime.Serialization;
```

Comment:
Turns out this is all we are using the `Microsoft.NETCore.Portable.Compatibility` dependency for.